requirements-dev.txt

```diff
@@ -31,6 +31,7 @@ django-password-validators==1.7.0
 django-csp==3.7
 djangorestframework==3.14.0
 django-tagulous==1.3.3
+drf-standardized-errors==0.12.4
```

Comment:
If this package is already included in `requirements.txt` then it's not needed here anymore, right?

Reply:
Some commands/environments only use `requirements-dev.txt`, so at the moment it is required. I guess we can refactor that in a future PR.

Reply:
Some commands/environments only use `requirements-dev.txt`, so at the moment it is required. I guess we can refactor that in a future PR.

Reply:
Some commands/environments only use `requirements-dev.txt`, so at the moment it is required. I guess we can refactor that in a future PR.